backend/minors/CGS.yaml

```diff
@@ -0,0 +1,224 @@
+---
+type: Minor
+name: Cognitive Science
+code: CGS
+degree:
+urls:
+- https://cogsci.princeton.edu/certificate
+description: Cognitive science is the interdisciplinary study of the 
+mind, brain, and intelligent systems, drawing predominantly on research 
+from psychology, philosophy, computer science, linguistics, neuroscience, 
+and anthropology. The interdisciplinary character of cognitive science is 
+reflected in its reliance on many levels of analysis and its employment of 
+a variety of methodologies appropriate to each level. The goal of cognitive 
+\science is to integrate the insights from multiple disciplines and modes of
+research into a unified scientific account of minds – be they human or 
+non-human, natural or artificial – and their place in nature and society.
+contacts:
+- type: Program Director
+  name: Anna Colasante
+  email: anna@princeton.edu
+req_list:
+  - name: Total Courses
+    explanation: Students are required to take five courses in cognitive science. At most, two courses counted towards the minor that are also counted towards the major. At most, one course with a primary course listing from the student's department of concentration unless permission is obtained from the faculty director."
+    min_needed: 5
+    max_counted: 1
+    req_list:
+    - name: Required Courses
+      explanation: At least one course designated with an asterisk in the list below, indicating that it is a CogSci Minor Required Course (CMRC);
+      min_needed: 1
+      max_counted: 5
+      course_list:
+        - CHV 300
+        - CGS 322
+        - CGS 254
+        - CGS 255
+        - COS 360
+        - LIN 309
+        - MUS 248
+        - NEU 306 
+        - PHI 207
+        - PHI 322
+        - PSY 248
+        - PSY 254
+        - PSY 255
+        - PSY 306
+        - PSY 309
+        - PSY 315
+        - PSY 333
+        - PSY 360
+        - PSY 454
```

Comment:
Let's make `max_counted` 1 here and double counting true. Make sure that courses here are in the other categories